### ISSUE TITLE
Add support for Recaptcha 2

### ISSUE THREAD

**Issue by ste93cry** [excelwebzone/EWZRecaptchaBundle#66]
It would be cool if you could add support for Recaptcha 2, which is far better than the previous one in both usability and security.

**Comment by garak** [comment]
:+1:

**Comment by Hoannv** [comment]
+1

**Comment by breitsmiley** [comment]
+1

**Comment by garak** [comment]
https://github.com/Bee-Lab/BeelabRecaptcha2Bundle

**Comment by ste93cry** [comment]
nice, but why didn't you just created a PR instead of making a new bundle?

**Comment by garak** [comment]
Because I don't need old version of Recaptcha

**Comment by roukmoute** [comment]
It is not already use in https://github.com/excelwebzone/EWZRecaptchaBundle/blob/master/Validator/Constraints/IsTrueValidator.php#L74 ?

**Comment by ste93cry** [comment]
no, the check is done with own methods instead of relying on the official Google library

**Comment by sebastianlp** [comment]
Is there a way to validate required recaptcha?
I'm setting this
            ->add('recaptcha', 'ewz_recaptcha', [
                'label' => false,
                'mapped' => false,
                'required' => true,
                'constraints' => [
                    new Required()
                ]
            ])

and doesn't seems to be working.
